src/Compilers/CSharp/Portable/Symbols/AliasSymbol.cs

```diff
@@ -73,7 +76,9 @@ private AliasSymbol(Binder binder, SyntaxToken aliasName)
         }
 
         internal AliasSymbol(Binder binder, UsingDirectiveSyntax syntax)
+#nullable disable // Can 'syntax.Alias' be null here? https://github.com/dotnet/roslyn/issues/39166
```

Comment:
No it cannot. There are only two call sites and they both validate that it's non-null.

Reply:
Resolved by passing the non-null arguments.